lanl/podman-quadlets/inventory/group_vars/ochami/packages.yaml

```diff
@@ -8,3 +8,4 @@ ochami_package_dependencies:
   - python3-requests
   - python3-yaml
   - python3-jwt
+  - ipxe-bootimgs-x86
```

Comment:
Do we need this package anymore? The bootloaders should be included in the CoreDHCP container as of coresmd v0.1.0.